practice_06.28.2019_delegates/Gender.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace practice_06._28._2019_delegates.Task2_DeferedInitialization
+{
+    public enum Gender
+    {
+        Male,
```

Comment:
Please don't forget to add integer values for each item.

Reply:
fixed